e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/report-reconciliation/generate-keying-sheet.spec.js

```diff
@@ -173,10 +173,8 @@ context('PDC_RECONCILE users can generate keying data', () => {
     const expectedPrincipalBalanceAdjustment = '128,000.00'; // 200000 - (180000 * 0.4), DECREASE
 
     /**
-     * Fixed fee adjustments are currently turned off and so should always be
-     * zero, which is displayed as a dash.
-     *
-     * TODO FN-3639: update with new calculation requirements.
+     * The fixed fee is not being used and so is always zero
+     * which is displayed as a dash
      */
     const expectedFixedFeeAdjustment = '-';
```

Comment:
is it worth checking with Clara if this should totally be removed since it will not be used

Reply:
We could ask, but it is out of scope for this ticket and there is a ticket in the backlog to cover redesign of the keying sheet after James Hall proposal decision is made so I'm not sure if it is worth doing anything intermediary?